authui/src/otpInput.ts

```diff
@@ -0,0 +1,119 @@
+import { Controller } from "@hotwired/stimulus";
+
+export class OtpInputController extends Controller {
+  static targets = ["input", "submit", "textContainer"];
+
+  declare readonly inputTarget: HTMLInputElement;
+  declare readonly submitTarget: HTMLButtonElement;
+  declare readonly textContainerTarget: HTMLElement;
+
+  value: string = "";
+  maxLength: number = 6;
```

Comment:
Currently use input maxlength to init, or should I be explicit and use values instead?

Reply:
using maxlength looks good to me.